src/main/java/com/getindata/connectors/http/internal/table/lookup/JavaNetHttpPollingClient.java

```diff
@@ -27,6 +27,10 @@
 @Slf4j
 public class JavaNetHttpPollingClient implements PollingClient<RowData> {
 
+    public static final String DEFAULT_REQUEST_MAX_RETRIES = "3";
+
+    public static final String DEFAULT_REQUEST_RETRY_TIMEOUT_MS = "1000";
```

Comment:
I suggest calling this DEFAULT_REQUEST_RETRY_INTERVAL_MS as it is not a timeout.